qa/tasks/thrashosds.py

```diff
@@ -153,6 +155,7 @@ def task(ctx, config):
     config['noscrub_toggle_delay'] = config.get('noscrub_toggle_delay', 2.0)
     # add default value for random_eio
     config['random_eio'] = config.get('random_eio', 0.0)
+    aggro = config.get('aggressive_pg_num_changes', true)
```

Comment:
`aggro = config.get('aggressive_pg_num_changes', "true")` could be the reason for the teuthology failures.